packages/sitecore-jss-react/src/components/Image.tsx

```diff
@@ -114,6 +114,38 @@ const getImageAttrs = (
   return newAttrs;
 };
 
+/**
```

Comment:
Can you provide more descriptions for each parameter? You have interfaces at the top with descriptions